cafebabel/static/css/styles.css

```diff
@@ -1033,6 +1033,93 @@ h1.edit input {
   text-align: center;
 }
 
+.create-article-page .select:nth-child(2) select {
+  margin-bottom: 5rem;
+}
+
+/* overwrite choices styles */
```

Comment:
I think these styles could be in their dedicated custom CSS, this way…

Reply:
but we can't overwrite their css within their css files, right ? so ?

Reply:
No, but you can create a new dedicated CSS with custom styles for that component.

Reply:
👍